### ISSUE TITLE
A. thalina QC

### ISSUE THREAD

**Issue by ndukler** [popgensims/stdpopsim#97]
Does anyone have QC code for the Arabidopsis models? I can also do it myself if there isn't any. As soon as the QC goes up I can add some documentation.

**Comment by andrewkern** [comment]
I think that @bernard-kim might have?

**Comment by petrelharp** [comment]
Brief question from the call today: any progress on this, @bernard-kim or @Chris1221?

**Comment by Chris1221** [comment]
Sorry, this was assigned to me but I got busy with other things in the last week. It's on my to-do list!

**Comment by Chris1221** [comment]
Sorry for being a bit dense, but I'm finding it a little hard to parse what needs to be done for the QC code. From what I can tell in the documentation, is it simply an independent implementation by a second developer which is checked for equality with the original model? If this is the case, I'll get started working on it right away, if not perhaps someone could give me a 2 second introduction in one of the upcoming calls.  Sorry again!

**Comment by ndukler** [comment]
Yep, that's basically it. The key bits are that you create it on a separate branch, then set up a PR with the QC model in the appropriate location and the appropriate test added. Look at the code I added in one of my commits to see an example.

**Comment by Chris1221** [comment]
Ah perfect, thanks very much Noah. I'll get on it today!

**Comment by ndukler** [comment]
If you can re-implement the model I can walk you through the rest. Also, if you have a disagreement with the production model for a non-trivial reason (trivial: match the order of events even if they could be swapped, pop. indices, etc.) you should talk to the person who implemented the production model and resolve it there if possible.

**Comment by Chris1221** [comment]
@arundurvasula Sorry to ping you, but I'm just wondering if you used the actual output of MSMC to build the model implemented here? I'm unfortunately unable to find the numbers that you used in either your paper or the supplement. (sorry!)

**Comment by arundurvasula** [comment]
Hey Chris, sorry about the confusion! 

Yes, I used the output of MSMC for the model. These numbers come from randomly sampling one of the pairs used in Figure 5 of that paper from the South Middle Atlas region (the samples are Khe32 and IFr4).

I attached the output of MSMC (`final.txt`) as well as the file converted to population size and years (`popsize.txt`) using _A. thaliana_ parameters (mutation rate = 7e-9; 1 generation per year). 

Let me know if you have any other questions!

[35608x35610.final.txt](https://github.com/popgensims/stdpopsim/files/3424419/35608x35610.final.txt)
[35608x35610.popsize.txt](https://github.com/popgensims/stdpopsim/files/3424420/35608x35610.popsize.txt)

**Comment by Chris1221** [comment]
That's awesome, thanks Arun! So sorry to make you search around and find those files. I appreciate the help, I'll get to work on reimplementing the models.
